src/main/java/com/glodblock/github/client/gui/GuiSuperStockReplenisher.java

```diff
@@ -0,0 +1,120 @@
+package com.glodblock.github.client.gui;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import net.minecraft.entity.player.InventoryPlayer;
+import net.minecraft.inventory.Slot;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ResourceLocation;
+
+import org.lwjgl.opengl.GL11;
+
+import com.glodblock.github.FluidCraft;
+import com.glodblock.github.client.gui.container.ContainerSuperStockReplenisher;
+import com.glodblock.github.common.item.ItemFluidPacket;
+import com.glodblock.github.common.tile.TileSuperStockReplenisher;
+import com.glodblock.github.network.CPacketInventoryAction;
+
+import appeng.api.storage.data.IAEItemStack;
+import appeng.api.storage.data.IAEStack;
+import appeng.client.gui.AEBaseGui;
+import appeng.container.AEBaseContainer;
+import appeng.container.slot.SlotFake;
+import appeng.core.localization.GuiText;
+import appeng.helpers.InventoryAction;
+import appeng.util.item.AEItemStack;
+
+public class GuiSuperStockReplenisher extends AEBaseGui {
+
+    private static final ResourceLocation TEX_BG = FluidCraft.resource("textures/gui/superStockReplenisher.png");
+    private final ContainerSuperStockReplenisher cont;
+    private final TileSuperStockReplenisher tile;
+    private Map<Integer, IAEStack<?>> list = new HashMap<>();
+
+    public GuiSuperStockReplenisher(InventoryPlayer ipl, TileSuperStockReplenisher tile) {
+        super(new ContainerSuperStockReplenisher(ipl, tile));
+        this.tile = tile;
+        this.cont = (ContainerSuperStockReplenisher) inventorySlots;
```

Comment:
Unused `cont` and `tile`.
```suggestion
    private Map<Integer, IAEStack<?>> list = new HashMap<>();

    public GuiSuperStockReplenisher(InventoryPlayer ipl, TileSuperStockReplenisher tile) {
        super(new ContainerSuperStockReplenisher(ipl, tile));
```